src/main/java/com/github/vkorobkov/jfixtures/domain/Row.java

```diff
@@ -17,15 +18,44 @@
     public Row(String name, Map<String, ?> columns) {
         this.name = name;
         this.columns = Collections.unmodifiableMap(
-            CollectionUtil.mapValues(columns, Value::of)
+                CollectionUtil.mapValues(columns, Value::of)
         );
     }
 
     public Row columns(Map<String, ?> toMerge) {
         val merged = CollectionUtil.merge(
-            columns,
-            CollectionUtil.mapValues(toMerge, Value::of)
+                columns,
+                CollectionUtil.mapValues(toMerge, Value::of)
         );
         return new Row(name, merged);
     }
+
+    public Row columns(Object... keyValuePairs) {
+        int pairsLength = keyValuePairs.length;
+
+        if (pairsLength > 1 && pairsLength % 2 != 0) {
+            throw new IllegalArgumentException(
+                    "Parameter <keyValuePairs> is expected to have odd length since it represents key/value pairs"
+            );
+        }
+
+        Map<String, Object> keyValueMap = new LinkedHashMap<>(pairsLength / 2);
+
+        if (pairsLength > 1) {
```

Comment:
I think even if we remove this `if` then `for` cycle does zero iteration in case `pairsLength == 0` so the condition is redundant